tests/debugging/exploration/test_bootstrap.py

```diff
@@ -44,8 +45,13 @@
 No lines found
 """
 
+if "PYTHONPATH" in os.environ:
```

Comment:
It looks like this function:
https://github.com/DataDog/dd-trace-py/blob/1.x/tests/internal/test_module.py#L16

Could we use that function in this test to avoid duplicate code? WDYT?

Reply:
because we need to pass `__file__` we would have to reimplement that function in this module unfortunately. Perhaps we can do a follow-up where we make a more generic helper?